agent/config/config.go

```diff
@@ -142,6 +142,10 @@ const (
 	defaultCgroupCPUPeriod = 100 * time.Millisecond
 	maximumCgroupCPUPeriod = 100 * time.Millisecond
 	minimumCgroupCPUPeriod = 8 * time.Millisecond
+
+	// DefaultContainerMetricsPublishInterval is the default interval that we publish
+	// metrics to the ECS telemetry backend (TACS)
+	DefaultContainerMetricsPublishInterval = 20 * time.Second
```

Comment:
why is this present in config package? do we foresee customers setting this value?

Reply:
we don't but it seemed like the most logical place to put global constants, we have other constants in this package that are not controllable by the customer so there's precedence for it, whether right or wrong.